src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -53,7 +55,8 @@
  *
  */
 public abstract class CommandLineProgram implements CommandLinePluginProvider {
-    protected final Logger logger = LogManager.getLogger(this.getClass());
+
+    protected final Logger logger = LogManager.getLogger(CommandLineProgram.class);
```

Comment:
In general we use a static logger and explicitly specify the class, this particular logger is the single exception.  I'm not sure it makes sense to do the halfway change here where it remains non-static but doesn't adapt to the actual class.